src/Alert/Alert.stories.js

```diff
@@ -150,3 +203,43 @@ NoDismiss.args = {
   hasDismiss: false,
 };
 NoDismiss.storyName = 'No Dismiss Button';
+
+export const ErrorSummaryBox = AlertTemplate.bind({});
+ErrorSummaryBox.args = {
+  children: errorBody,
+  type: 'errorSummary',
+  label: '',
+  alignContent: 'top',
+  classes: {
+    wrapper: 'su-w-full su-max-w-700',
+    container: 'su-px-18 su-pt-18 su-pb-26',
```

Comment:
```suggestion
    container: 'su-rs-px-0 su-rs-pt-0 su-rs-pb-1',
```
Please use responsive spacing for these 😃